components/camel-huaweicloud-smn/src/main/docs/hwcloud-smn-component.adoc

```diff
@@ -71,13 +71,15 @@ with the following path and query parameters:
 |===
 
 
-=== Query Parameters (13 parameters):
+=== Query Parameters (14 parameters):
 
 
 [width="100%",cols="2,5,^1,2",options="header"]
 |===
 | Name | Description | Default | Type
 | *authKey* (producer) | *Required* Authentication key for the cloud user |  | String
+| *endpoint* (producer) | Fully qualified smn service url. Carries higher precedence than region parameter based client initialization|  | String
```

Comment:
This is autogenerated, you need to update the description on the option, not change the table directly, once you build the module, it will be updated.

Reply:
@oscerd - thanks for the note. Updates done to the migration guide and also mentioned the same in component documentations.

Reply:
Thanks @oscerd for the tip. Have checked in as per guidelines. We loved the entire documentation framework :)